src/components/display/fewlines/LoginsOverview/LoginsOverview.tsx

```diff
@@ -36,277 +13,61 @@ type LoginsOverviewProps = {
-      case IdentityTypes["NAVER"]:
-        return SOCIAL_IDENTITIES_ICONS.NAVER;
-      case IdentityTypes["PAYPAL"]:
-        return SOCIAL_IDENTITIES_ICONS.PAYPAL;
-      case IdentityTypes["STRAVA"]:
-        return SOCIAL_IDENTITIES_ICONS.STRAVA;
-      case IdentityTypes["VKONTAKTE"]:
-        return SOCIAL_IDENTITIES_ICONS.VKONTAKTE;
-      case IdentityTypes["MICROSOFT"]:
-        return SOCIAL_IDENTITIES_ICONS.MICROSOFT;
-      case IdentityTypes["DECATHLON"]:
-        return SOCIAL_IDENTITIES_ICONS.DECATHLON;
-      default:
-        return <React.Fragment />;
-    }
-  };
-
-  let emailList: Identity[];
-  let phoneList: Identity[];
-
-  hideSecondaryEmails
-    ? (emailList = sortedIdentities.emailIdentities.filter(
-        (identity) => identity.primary,
-      ))
-    : (emailList = sortedIdentities.emailIdentities);
-
-  hideSecondaryPhones
-    ? (phoneList = sortedIdentities.phoneIdentities.filter(
-        (identity) => identity.primary,
-      ))
-    : (phoneList = sortedIdentities.phoneIdentities);
-
   const {
     emailIdentities,
     phoneIdentities,
     socialIdentities,
   } = sortedIdentities;
 
-  return (
-    <Container>
-      <IdentitySection>
-        <Timeline />
-        <h2>Email addresses</h2>
-        <ShadowBox>
-          {emailIdentities.length === 0 ? (
-            <NoIdentitiesBox>No emails added yet.</NoIdentitiesBox>
-          ) : (
-            emailList.map((email: Identity) => {
-              return (
-                <div key={email.value}>
-                  <BoxedLink
-                    primary={email.primary}
-                    status={email.status}
-                    href={{
-                      pathname: "/account/logins/[type]/[id]",
-                      query: {
-                        type: email.type.toLowerCase(),
-                        id: email.id,
-                      },
-                    }}
-                  >
-                    <p>{email.value}</p>
-                    <RightChevron />
-                  </BoxedLink>
+  const IDENTITIES_SECTION_CONTENT = {
+    EMAIL: {
+      title: "Email addresses",
+      identitiesList: emailIdentities,
+      displayListMethod: displayIdentity,
+      disableClick: false,
+      noIdentityMessage: "No email added yet.",
+      addNewIdentityMessage: "add new email address",
+      hideSecondaryByDefault: true,
+    },
+    PHONE: {
+      title: "Phone numbers",
+      identitiesList: phoneIdentities,
+      displayListMethod: displayIdentity,
+      disableClick: false,
+      noIdentityMessage: "No phone number added yet.",
+      addNewIdentityMessage: "add new phone number",
+      hideSecondaryByDefault: true,
+    },
+    SOCIAL_LOGINS: {
+      title: "Social logins",
+      identitiesList: socialIdentities,
+      displayListMethod: displaySocialLogins,
+      disableClick: true,
+      noIdentityMessage: "No social logins added yet.",
+      hideSecondaryByDefault: false,
+    },
+  };
```

Comment:
This should be outside the component if we want to use it for integration tests.

Reply:
For now I can't because `identitiesList` value are only known within the component (thanks to `sortedIdentities`).
A solution would be to move this object outside and to append `identitiesList` afterwards inside the component.
Do you want me to do so?

Reply:
Well, `IDENTITIES_SECTION_CONTENT` shouldn't have dynamic data (i.e. Identities), because we want to use this object in our tests to make refactoring of the content easier (i.e. changing the strings in only one place).

Reply:
Ok, is it good that way ? f22b2c4